src/einsteinpy/symbolic/schouten.py

```diff
@@ -0,0 +1,108 @@
+from einsteinpy.symbolic.ricci import RicciScalar, RicciTensor
+from einsteinpy.symbolic.riemann import RiemannCurvatureTensor
+from einsteinpy.symbolic.tensor import BaseRelativityTensor, _change_config
+
+
+class SchoutenTensor(BaseRelativityTensor):
+    """
+    Inherits from ~einsteinpy.symbolic.tensor.BaseRelativityTensor .
+    Class for defining Schouten Tensor
+    """
+
+    def __init__(self, arr, syms, config="ll", parent_metric=None):
+        """
+        Constructor and Initializer
+
+        Parameters
+        ----------
+        arr : ~sympy.tensor.array.dense_ndim_array.ImmutableDenseNDimArray or list
+            Sympy Array or multi-dimensional list containing Sympy Expressions
+        syms : tuple or list
+            Tuple of crucial symbols denoting time-axis, 1st, 2nd, and 3rd axis (t,x1,x2,x3)
+        config : str
+            Configuration of contravariant and covariant indices in tensor. 'u' for upper and 'l' for lower indices. Defaults to 'll'.
+        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor
+            Corresponding Metric for the Schouten Tensor. Defaults to None.
+
+        Raises
+        ------
+        TypeError
+            Raised when arr is not a list or sympy Array
+        TypeError
+            syms is not a list or tuple
+        ValueError
+            config has more or less than 2 indices
+
+        """
+        super(SchoutenTensor, self).__init__(
+            arr=arr, syms=syms, config=config, parent_metric=parent_metric
+        )
+        self._order = 2
+        if not len(config) == self._order:
+            raise ValueError("config should be of length {}".format(self._order))
+
+    @classmethod
+    def from_metric(cls, metric):
+        """
+        Get Schouten tensor calculated from a metric tensor
+
+        Parameters
+        ----------
+        metric : ~einsteinpy.symbolic.metric.MetricTensor
+            Space-time Metric from which Christoffel Symbols are to be calculated
+
+        Raises
+        ------
+        ValueError
+            Raised when the dimension of the tensor is less than 3
+
+        """
+        if metric.dims >= 3:
+            t_riemann = RiemannCurvatureTensor.from_metric(metric)
+            t_ricci = RicciTensor.from_riemann(t_riemann, parent_metric=None)
+            r_scalar = RicciScalar.from_riccitensor(t_ricci, parent_metric=None)
+            dims = metric.dims
+            t_schouten = (
+                1
+                / (dims - 2)
+                * (
+                    t_ricci.tensor()
+                    - 1 / (2 * (dims - 1)) * r_scalar._expr * metric.tensor()
```

Comment:
Can you replace `t_schouten` with this?
`(t_ricci.tensor() - (r_scalar.expr*metric.tensor()/(2*(dims-1))))/(dims-2)`

Reply:
Sorry for me being too fussy 😞

Reply:
jaja Don't worry! :)